drivers/wdm/audio/backpln/portcls/pin_wavert.cpp

```diff
@@ -305,35 +305,46 @@ CPortPinWaveRT::DeviceIoControl(
     IoStack = IoGetCurrentIrpStackLocation(Irp);
 
 
-    if (IoStack->Parameters.DeviceIoControl.IoControlCode == IOCTL_KS_PROPERTY)
+    switch (IoStack->Parameters.DeviceIoControl.IoControlCode)
     {
-       return HandleKsProperty(Irp);
+        case IOCTL_KS_PROPERTY:
+            return HandleKsProperty(Irp);
+            break;
+		
+        case IOCTL_KS_ENABLE_EVENT:
+            /* FIXME UNIMPLEMENTED */
+            UNIMPLEMENTED_ONCE;
```

Comment:
`UNIMPLEMENTED_ONCE` everywhere now... Why (not).

Reply:
For.. debugging purposes?

Reply:
Hum, which debugging purposes changing the macro accomplishes?